geometry_msgs/Point center # Point defining the center of a rectangular surface.
geometry_msgs/Vector3 edge_1 # Vector defining the first edge of a rectangular surface. The norm of the vector is half the length of the edge.
geometry_msgs/Vector3 edge_2 # Vector defining the second edge of a rectangular surface. The norm of the vector is half the length of the edge.
